db/schema.rb

```diff
@@ -1,6 +1,7 @@
 ActiveRecord::Schema.define(version: 20160719122949) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+<<<<<<< HEAD
```

Comment:
That's a git marking of where conflict begins. Deal with the conflicts before pushing.